mantle/kola/tests/ignition/qemufailure.go

```diff
@@ -101,3 +118,112 @@ func ignitionFailure(c cluster.TestCluster) error {
 		return nil
 	}
 }
+
+// get current path and create tmp dir
+func GetTempdir(dirname string) (string, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		return "", err
+	}
+	path := filepath.Join(dir, dirname)
+	return path, nil
+}
+
+func bootfsFailure(c cluster.TestCluster) error {
+	// We can't create files in / due to the immutable bit OSTree creates, so
+	// this is a convenient way to test Ignition failure.
+
+	failConfig, err := conf.EmptyIgnition().Render(conf.FailWarnings)
+	if err != nil {
+		return errors.Wrapf(err, "creating empty config")
+	}
+
+	// Config which theoretically aims to reference fakeboot disk
+	fakeConfig, err := conf.EmptyIgnition().Render(conf.FailWarnings)
+	if err != nil {
+		return errors.Wrapf(err, "creating empty config")
+	} else {
+
+		tempDir, err := GetTempdir("tmp")
+		if err != nil {
+			fmt.Println("Error getting temp directory:", err)
+		}
+		fmt.Println("Temp directory:", tempDir)
+		cmd := exec.Command("/bin/bash", "-c", fmt.Sprintf(`set -euo pipefail;
+				mkdir -p %s
+				truncate -s 1G fakeboot
+				mkfs.ext4 -L boot fakeboot
+			`, tempDir))
```

Comment:
Not against calling the `truncate` command, but it's not too much work I think to do this in native golang: you can [`os.Create()`](https://pkg.go.dev/os#Create) the file, and then [`Truncate()`](https://pkg.go.dev/os#File.Truncate) on the file.